src/api/models/classification.py

```diff
@@ -0,0 +1,155 @@
+    benthic_attribute = models.ForeignKey(
+        BenthicAttribute, related_name="labels", on_delete=models.CASCADE
+    )
+    growth_form = models.ForeignKey(GrowthForm, related_name="labels", on_delete=models.CASCADE)
+    description = models.TextField(null=True, blank=True)
+
+    class Meta:
+        unique_together = ("benthic_attribute", "growth_form")
+        db_table = "class_label"
+
+    def __str__(self):
+        return self.name
+
+    @property
+    def name(self):
+        return f"{self.benthic_attribute.name} {self.growth_form.name}"
+
+
+class Classifier(BaseModel):
+    name = models.CharField(max_length=50)
+    version = models.CharField(max_length=11)
+    patch_size = models.IntegerField(help_text="Number of pixels")
+    num_points = models.IntegerField(default=25)
+    description = models.TextField(max_length=1000, blank=True)
+    labels = models.ManyToManyField(Label, related_name="classifiers")
+
+    class Meta:
+        db_table = "class_classifier"
+
+
+class Image(BaseModel):
+    collect_record_id = models.UUIDField()
+    observation = models.ForeignKey(
+        ObsBenthicPhotoQuadrat, null=True, blank=True, on_delete=models.SET_NULL
+    )
+
+    image = models.ImageField(upload_to="", storage=select_image_storage)
+    original_image_checksum = models.CharField(max_length=64, blank=True, null=True)
+    thumbnail = models.ImageField(upload_to="", storage=select_image_storage, null=True, blank=True)
+
+    name = models.CharField("Name", max_length=200, blank=True, null=True)
+    photo_timestamp = models.DateTimeField(null=True, blank=True)
+    location = models.PointField(srid=4326, blank=True, null=True)
+    comments = models.TextField(max_length=1000, blank=True, null=True)
+    data = models.JSONField("Data", null=True, blank=True)
+
+    class Meta:
+        db_table = "class_image"
+
+    def __str__(self):
+        return f"{self.name} - {self.photo_timestamp}"
+
+    @property
+    def collect_record(self):
+        return CollectRecord.objects.get_or_none(id=self.collect_record_id)
+
+    @property
+    def project(self):
+        cr = self.collect_record
+        if cr:
+            return cr.project
+
+        obs = self.observation
+        return obs.benthic_photo_quadrat_transect.quadrat_transect.sample_event.site.project
+
+
+class Point(BaseModel):
+    row = models.IntegerField()
+    column = models.IntegerField()
+    point_number = models.IntegerField()
+    image = models.ForeignKey(Image, on_delete=models.CASCADE, related_name="points")
+
+    class Meta:
+        db_table = "class_point"
+
+
+class Annotation(BaseModel):
+    point = models.ForeignKey(Point, on_delete=models.CASCADE, editable=False)
+    label = models.ForeignKey(Label, related_name="annotations", on_delete=models.CASCADE)
+    classifier = models.ForeignKey(
+        Classifier, null=True, on_delete=models.CASCADE, related_name="annotations"
+    )
+    score = models.PositiveSmallIntegerField(default=0)
```

Comment:
We can set this as nullable but ...

```
Like a [PositiveIntegerField](https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.PositiveIntegerField), but only allows values under a certain (database-dependent) point. Values from 0 to 32767 are safe in all databases supported by Django.
```

Reply:
You mean the min/max constraint? We just need to handle 0-100. Or does this means we'd have to use a different field type because `PositiveSmallIntegerField` doesn't permit NULLs? If so maybe it's not worth changing -- we just need to know that a null `classifier` is what's important, and use that to distinguish; a 0 `score` could in theory be human, or not.